profile/README.md

```diff
@@ -11,6 +11,7 @@ Among other things, this page is used to share datasets and software projects.
 + [LA-ICPMS measurements of elemental concentration from floatglass](https://github.com/NetherlandsForensicInstitute/elemental_composition_glass)
 + [Body fluids from mRNA assay data in mixtures](https://github.com/NetherlandsForensicInstitute/body_fluids_mRNA)
 + [iPhone Health app data](https://github.com/NetherlandsForensicInstitute/iphone-health-app-data)
++ [Gunshot residue comparisons](https://github.com/NetherlandsForensicInstitute/gunshot-residue)
```

Comment:
The data are of the GSR, not necessarily of the comparisons I think? Maybe 'SEM measurements of gunshot residue samples', also to be more consistent with the other descriptions?

Reply:
Edited